seqr/utils/elasticsearch/es_search.py

```diff
@@ -1149,21 +1155,21 @@ def _pathogenicity_filter(pathogenicity):
         clinvar_clinical_significance_terms = set()
         for clinvar_filter in clinvar_filters:
             clinvar_clinical_significance_terms.update(CLINVAR_SIGNFICANCE_MAP.get(clinvar_filter, []))
-        pathogenicity_filter = Q('terms', clinvar_clinical_significance=list(clinvar_clinical_significance_terms))
+        pathogenicity_filter = Q('terms', clinvar_clinical_significance=sorted(list(clinvar_clinical_significance_terms)))
 
     if hgmd_filters:
         hgmd_class = set()
         for hgmd_filter in hgmd_filters:
             hgmd_class.update(HGMD_CLASS_MAP.get(hgmd_filter, []))
 
-        hgmd_q = Q('terms', hgmd_class=list(hgmd_class))
+        hgmd_q = Q('terms', hgmd_class=sorted(list(hgmd_class)))
```

Comment:
The reason for adding `sorted()` is to make the test result has a predictable order.